lib/jekyll-relative-links/generator.rb

```diff
@@ -78,7 +85,9 @@ def url_for_path(path)
     end
 
     def potential_targets
-      @potential_targets ||= (site.pages + site.static_files)
+      @potential_targets ||= begin
+        site.pages + site.static_files + (collections? ? site.documents : [])
```

Comment:
Should you be able to link *to* a collection document regardless of if collections are enabled?

Reply:
hm, coming to think about it, the collections option could be enabled by default for this plugin. Are there meaningful ways to use relative links inside a post/collection that would be broken by this change?

Reply:
in any case, `site.docs_to_write` can be part of the potential targets.